strings/build.gradle

```diff
@@ -34,7 +33,7 @@ android {
 
 dependencies {
     implementation Dependencies.androidx_core_ktx
-    implementation Dependencies.androidx_appcompat
+    api Dependencies.androidx_appcompat
```

Comment:
Why? Do you want to use an internal library of `androidx_appcompat` or something?

Reply:
This is a weird one that I should have left a comment here about. Try switching back to `implementation` and you'll see that `CaptureSelfieActivity` will fail to build as it won't have access to methods in `AppCompatActivity`. Surprisingly, to extend a class you need to have that classes supertype(s) in the classpath as well. 

You can set up an experiment to get a more obvious error I got where you have 3 modules: the first contains an `interface` the second depends on the first and contains an `abstract` class that extends that interface and the third depends on the second and extends the abstract class. When you try to compile you get an error:

```
Supertypes of the following classes cannot be resolved. Please make sure you have the required dependencies in the classpath:
    class second.Abstract, unresolved supertypes: first.Interface
```

So for a module to extend `LocalizedActivity` it also needs `AppCompatActivity` in the classpath. To make sure that's always the case, I expose `appcompat` as an `api` dependency. I'll add a comment to make that explicit.